src/postgres/commands/PostgresQueryWizard/FunctionQuerySteps/IPostgresFunctionQueryWizardContext.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { QuickPickItem } from "vscode";
+import { IPostgresQueryWizardContext } from "../IPostgresQueryWizardContext";
+
+export interface IPostgresFunctionQueryWizardContext extends IPostgresQueryWizardContext {
+    name: string;
```

Comment:
wizard context properties should almost always be optional with `?` since they don't exist until after the prompt step. That means you'll have to use a decent amount of `nonNullProp` whenever you want to access it, but it does make it easer to find bugs in telemetry if we have any steps out of order